pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -165,6 +230,12 @@ func (v *VPCConnectivity) computeAllowedStatefulConnections() {
 
 	for src, connsMap := range v.AllowedConnsCombined {
 		for dst, conn := range connsMap {
+			// iterate pairs (src,dst) with conn as allowed connectivity, to check stateful aspect
+			if v.isConnExternalThroughFIP(src, dst) {
+				updateAllowedConnsMap(v.AllowedConnsCombinedStateful, src, dst, conn)
+				continue
+			}
```

Comment:
This might be an IBM-specific thing. For now maybe just add a comment or a TODO.

Reply:
done